components/clarinet-deployments/src/lib.rs

```diff
@@ -322,11 +321,7 @@ pub async fn generate_default_deployment(
                     ))
                 }
             };
-            queue.push_front((
-                contract_id,
-                StacksEpochId::Epoch21,
-                ClarityVersion::Clarity2,
-            ));
+            queue.push_front((contract_id, Some(DEFAULT_CLARITY_VERSION)));
```

Comment:
Do we not need the epoch anymore with this change?

Reply:
@obycode It was already not need, it was only passed from items to items in the queue, but except for that, the value was never used

Reply:
Ah, I see. Thanks!